src/WebUI/Pods/PodOverview.scss

```diff
@@ -0,0 +1,11 @@
+@import "../Common/Common.scss";
+
+.pod-overview-key {
+    padding-left: 8px;
+    color: $secondary-text;
+}
+
+.kube-list-content .bolt-header.bolt-card-header {
```

Comment:
this is specific to platform, if they change we have a breakage, shall we keep this in a file.

.bolt-header.bolt-card-header

Reply:
I have moved all platform css to a common file Webplatform.scss